src/PKSim.Core/Services/ExpressionProfileUpdater.cs

```diff
@@ -144,16 +144,6 @@ public void SynchroniseSimulationSubjectWithExpressionProfile(ISimulationSubject
          var (sourceMolecule, sourceIndividual) = expressionProfile;
          // ExpressionProfile => SimulationSubject, we want to make sure that the parameters in simulation subject are linked to their expression profile origin parameters
          synchronizeExpressionProfiles(sourceMolecule, sourceIndividual, moleculeInIndividual, simulationSubject, updateParameterOriginId: true);
-
-         //Once the synchronization was performed, apply changes to simulation subject molecules based on disease state
-         updateMoleculeParametersForDiseaseState(simulationSubject, moleculeInIndividual);
-      }
-
-      private void updateMoleculeParametersForDiseaseState(ISimulationSubject simulationSubject, IndividualMolecule moleculeInIndividual)
```

Comment:
@Yuri05 @rwmcintosh This was a theoretical  implementation but no real implementation until now. This did not work